app/templates/preprints/view.hbs

```diff
@@ -3,74 +3,51 @@
-                    <div> {{!IFRAME DIV}}
-                        <iframe src={{concat model.renderUrl '?url=' model.downloadLink}} display="block"
-                                width=650 height=700></iframe>
-                    </div> {{!END IFRAME DIV}}
-                    {{!TODO: EMBER OSF MFR }}
-
-                </div> {{!END MFR COL DIV}}
+            <div class="row p-l-md"> {{!PREPRINT INFO ROW}}
+                <div class="col-md-10">
+                    <h5>{{model.preprint.authors}}</h5>
+                    <h6>Added on {{moment-format model.preprint.date}}</h6>
+                </div>
+                <div class="col-md-2">
+                    <h6 class="m-t-lg"><span>Provider: {{#link-to "index"}}SocArXiv{{/link-to}} </span><img src="preprints/assets/img/logo_socarxiv.jpg" width="30"></h6>
+                </div>
+            </div> {{!END PREPRINT INFO ROW}}
+            <div class="row">
+                <hr color="#C7FFC7">
+            </div>
+            <div class="row p-md"> {{!CONTENT ROW}}
+                <div class="col-md-7"> {{!LEFT COL DIV}}
+                    {{#if model.supplement}}
+                        {{supplementary-file-browser primaryFile=model.primary files=model.supplement}}
+                    {{/if}}
+                </div> {{!END LEFT COL DIV}}
                 <div class="col-md-5"> {{!RIGHT SIDE COL}}
-                    {{!DOWNLOAD BUTTON}}
-                    <br>
-                    <div class="m-b-md p-md osf-box-lt" id="download_project">
-                        <a class="btn btn-default" href={{model.downloadLink}}>
-                            <span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span>
-                            Download preprint
-                        </a>
-                    </div>
-                    <br>
-                    {{!PROJECT BUTTON}}
-                    <div class="m-b-md p-md osf-box-lt" id="view_project">
-                        <div class="col-md-8"> {{!TEXT COL 8}}
-                            <p>The project for this paper is available on the Open Science Framework</p>
-                        </div> {{!END TEXT COL 8}}
-                        <div class="col-md-4"> {{!BUTTON COL 4}}
-                            <a type="button" class="btn btn-primary" href={{concat model.baseUrl model.project}}>
-                                View project
-                            </a>
-                        </div> {{!END BUTTON COL 4}}
-                    </div>
-                    <br>
+                    <div class="p-md osf-box-lt" id="download_project">{{!SHARE ROW}}
+                        <i class="fa fa-download" aria-hidden="true"></i>
+                        <a href={{model.downloadLink}}>Download</a>
+                        <i class="fa fa-edit" aria-hidden="true"></i>
+                        {{#link-to "preprints.edit" model.preprint.id}}Edit{{/link-to}}
```

Comment:
If you pass `preprint` instead of `preprint.id`, it will use the model given. Basically, it saves the page from making another request.

(the catch, of course, is that the model you pass has to be what the model hook would have returned normally! Eg it doesn't work so well if the model hook would have returned an RSVP.hash)